src/main/java/edu/ie3/datamodel/models/input/system/characteristic/CharacteristicInput.java

```diff
@@ -121,24 +113,21 @@ public SortedSet<CharacteristicPoint<A, O>> getPoints() {
   public String deSerialize() {
```

Comment:
Ok maybe we can work on the naming here?

"In computing, serialization (or serialisation) is the process of translating a data structure or object state into a format that can be stored (e.g. files in secondary storage devices, data buffers in primary storage devices) or transmitted (e.g. data streams over computer networks) and reconstructed later (possibly in a different computer environment)."

"The opposite operation, extracting a data structure from a series of bytes, is deserialization, (also called unserialization or unmarshalling)."

(both from Wikipidia)

If we consider our Java objects here as the objects, and the output as a String as the "data [...] format that can be stored [...] and reconstructed later", this method should be called `serialize()`. 
But maybe we can also find a name that brings less heavy connotations with it, e.g. `export`...

@t-ober @danielfeismann

Reply:
I would be fine with just going for `serialize()` as this is actually pretty fitting to what we do here.